oat-sa/extension-tao-testqti
# Feature BSA 208 - Update de nl translations

**New fixes related to this PR**

https://github.com/oat-sa/extension-tao-testqti/pull/1890

**Related to this issue**

https://oat-sa.atlassian.net/browse/BSA-208

**Description**

Apply German and Dutch translation.

**How to test**

- Switch to branch
- Run `php ./tao/scripts/taoUpdate.php`
- Create a test with a review panel, different tools, timers
- Create test takers with language German and Dutch
- Run delivery and check labels
